cpu/cc2538/startup-gcc.c

```diff
@@ -30,22 +30,23 @@
  *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 /**
- * \addtogroup cc2538dk
+ * \addtogroup cc2538
  * @{
  *
  * \file
- * Startup code for the cc2538dk platform, to be used when building with gcc
+ * Startup code for the cc2538 chip, to be used when building with gcc
  */
 #include "contiki.h"
 #include "reg.h"
 #include "flash-cca.h"
 #include "sys-ctrl.h"
-#include "uart.h"
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
 extern int main(void);
 /*---------------------------------------------------------------------------*/
+#define WEAK_ALIAS(x) __attribute__ ((weak, alias(#x)))
```

Comment:
Ok. By comparison the other macros in `cc.h` are used throughout the `core/` tree and by some 30+ files (by a cursory count). 

In any case, if someone really wants to discuss adding CPU-specific stuff to `cc.h`, let's make a separate pull request with those changes instead of slowing down this one (which shouldn't need to touch any core files).